src/Sarif.FunctionalTests/v2/ConverterTestData/PREfast/AA_IndexAlias.cpp.xml.sarif

```diff
@@ -12,6 +12,12 @@
           "mimeType": "text/x-cpp"
         }
       },
+      "logicalLocations": {
+        "C6385_IndexAlias": {
+          "name": "C6385_IndexAlias",
+          "decoratedName": "?C6385_IndexAlias@@YAXH@Z"
+        }
+      },
```

Comment:
Yeah, that looks right. Strictly speaking you don't need `name` if it's the same as the key. I'll keep reading and decide if this is worth fixing.

Reply:
Yes, do fix this. The spec says:
>If the `logicalLocation` object describes a top-level logical location, and if the `name` property would equal the name of the property for which this object provides the value, then the name property **MAY** be absent.
So what you have here is legal SARIF, but our code should produce "best" SARIF, and I think it best not to emit unnecessary properties.

Reply:
@michaelcfanning This is another non-optimal behavior in the PREfast converter (not strictly a bug) where it populates `logicalLocation.name` when it doesn't need to. @EasyRhin0 please file.

Reply:
Fixed